remo/featuredrep/templates/featuredrep_list.html

```diff
@@ -103,6 +103,34 @@
       </table>
     </div>
   </div>
+
+  <div class="row">
+    <div class="large-12 columns pagination" id="featured-pagination">
+      {% if featured.has_previous() %}
+        <a class="prev" title="Previous page" data-tooltip
+           href="{{ '#'|urlparams(page=featured.previous_page_number(),
+                                  sort_key=sort_key, query=query) }}">
+          &laquo;
+        </a>
+      {% endif %}
+      <label for="page-select">Page: </label>
+      <select id="page-select" autocomplete="off">
+        {% for i in range(1, featured.paginator.num_pages + 1) %}
+          <option {% if i == featured.number %} selected {% endif %}
+                  value="{{ '#'|urlparams(page=i, sort_key=sort_key, query=query) }}">
+            {{ i }}
+          </option>
+        {% endfor %}
+      </select>
+      {% if featured.has_next() %}
+        <a class="next" title="Next page" data-tooltip
+           href="{{ '#'|urlparams(page=featured.next_page_number(),
+                                  sort_key=sort_key, query=query) }}">
+          &raquo;
+        </a>
+      {% endif %}
+    </div>
+  </div>
```

Comment:
By adding this to the `featuredrep` app, we have this block of code to three different apps. Maybe we should consider adding this in a separate file in the base app and import it where we need it. This way we will avoid duplicate code.